packages/sdk-components-react-radix/src/theme/tailwind-colors.ts

```diff
@@ -19,6 +19,15 @@ export const colors = {
   ring: "rgb(148, 163, 184)",
   mutedForeground: "rgb(100, 116, 139)",
   muted: `hsl(210 40% 96.1%)`,
+  primary: "rgb(15, 23, 42)",
+  primaryForeground: "hsl(210 40% 98%)",
+  destructive: "rgb(239, 68, 68)",
+  destructiveForeground: "rgb(248, 250, 252)",
+  accent: "rgb(241, 245, 249)",
+  accentForeground: "rgb(15, 23, 42)",
+  input: "rgb(226, 232, 240)",
+  secondary: "rgb(241, 245, 249)",
+  secondaryForeground: "rgb(15, 23, 42)",
```

Comment:
Maybe preserve hsl for everything?

Reply:
alpha for bg not working with hsl, some hsl -> rgb converter will be needed